src/components/container/App/App.js

```diff
@@ -1,14 +1,27 @@
+import React, { useState } from "react";
 import HeroSection from "../../components/HeroSection/HeroSection";
 import Navbar from "../../components/Navbar/Navbar";
 import FeaturedJobs from "../FeaturedJobs/FeaturedJobs";
 import PartnerBanner from "../PartnerBanner/PartnerBanner";
 import Footer from "../../components/Footer/Footer";
 import ReinforcementSection from "../../components/ReinforcementStatement/ReinforcementStatement";
 import "./App.scss";
+import MobileNavigation from "../../components/MobileNavigation/MobileNavigation";
+import NavbarMobileLogoBar from "../../components/NavbarMobileLogoBar/NavbarMobileLogoBar";
 
 function App() {
+  const [isClicked, setIsClicked] = useState(false);
```

Comment:
Would rename this to `isMobileMenuOpened` or something like that, to see what this state actually means

Reply:
fair enough!